### ISSUE TITLE
[Bug]: PDF Viewer (Chrome extension) is not working in Kiwi Browser

### ISSUE THREAD

**Issue by towfikomar** [mozilla/pdf.js#19116]
### Attach (recommended) or Link to PDF file

https://github.com/user-attachments/assets/f0bdb5d1-ab91-42f4-8c57-fa7fdd404da3

### Web browser and its version

Kiwi Browser 124.0.6327.4

### Operating system and its version

Android 11

### PDF.js version

4.6.129

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

Yes

### Steps to reproduce the problem

1. Open a PDF link 
2. It will download , will not open in PDF viewer extension

### What is the expected behavior?

PDF link should be open in the extension.

### What went wrong?

PDF gets downloaded not opening in the extension. 
In conclusion extension is not working.

**Comment by towfikomar** [comment]
v4.3.84 working in kiwi.
Maybe it's because of MV3 and Kiwi hasn’t been updated since April, 2024

**Comment by Snuffleupagus** [comment]
> v4.3.84 working in kiwi.
> Maybe it's because of MV3 and Kiwi hasn’t been updated since April, 2024

In that case it unfortunately seems unlikely to be something that we can "fix" here, since the Chrome addon was purposely updated to use Manifest Version 3, and please also note that the Kiwi browser is *not* listed as supported [in the FAQ](https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-support).
